GA-worksheet.md

```diff
@@ -26,21 +26,21 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
-
+The link tag is a link to something or a heading. It also makes a specific style.
 2. What is normalize.css?
-
+Normalize.css renders all elements more constantly and in line.
 3. Why do we need to add a padding property to the `<ul>` tag?
-
+We need to add padding property tothe ul so the bullets won't appear or be at the corner looking unprofessional.
```

Comment:
The padding around the <ul> is tricky -- there is default padding on the left for unordered lists to accommodate the bullet points, so adding padding to all sides makes this particular ul appear centered.